utils/vcsdata.go

```diff
@@ -1,7 +1,6 @@
 package utils
```

Comment:
This file still uses both "details" and "data".
See also the name of the file...